package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "fec",
+  "version": "1.0.0",
+  "description": "FEC project",
+  "main": "webpack.config.js",
+  "dependencies": {
+    "express": "^4.17.1",
+    "mysql": "^2.17.1",
+    "react-dom": "^16.8.6",
+    "react": "^16.8.6",
+    "webpack-cli": "^3.3.5"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.5.0",
+    "@babel/preset-env": "^7.5.2",
+    "@babel/preset-react": "^7.0.0",
+    "babel-loader": "^8.0.6",
+    "webpack": "^4.35.3"
+  },
+  "scripts": {
```

Comment:
I recommend alaphabetizing key-value pairs in an object unless you have a compelling reason to do so otherwise.